kotlin/feature-layer-feature-service/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since there is only one view, I think it makes sense to remove the constraint layout (and build.gradle dependency)

Reply:
Good idea!